lib/std/fifo.zig

```diff
@@ -474,6 +486,15 @@ test "LinearFifo" {
                 const n = fifo.read(&readBuf);
                 testing.expectEqual(@as(usize, 3), n); // NOTE: It should be the number of items.
             }
+            
+            // Test partial writes via OutStream API
+            if(T == u8 and (bt == .Static or bt == .Slice)) {
+                const data = [_]T{ 1, } ** 34;
+                testing.expectEqual(@as(usize, 32), try fifo.outStream().write(&data));
```

Comment:
Suggestion: Maybe rebase this onto master where the new naming convetions of `Reader` and `Writer` instead of `InStream` and `OutStream` have taken effect?